### PR TITLE
Benefit Eligibility Questionnaire

### PR DESCRIPTION
**Pull request by vetskrieg**
## Summary

This PR introduces a new form application that is disabled. The Benefit Eligibility Questionnaire is added to the root URL /benefit-eligibility-questionnaire. This is the first step in the development of the following Figma design.

[Figma](https://www.figma.com/design/DNy6D8ELfTQW1g3EK7yJu5/Mid-Fidelity-Prototype-for-v1?node-id=0-1&t=rl49LMm3Igw3gnJf-0)

Team: VES Transition

## Related issue(s)

[PTEMSVT-63](https://jira.devops.va.gov/browse/PTEMSVT-63)

## Testing done

Local testing of new benefit-eligibility-questionnaire application.

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |image|
| Desktop |        |image|

## What areas of the site does it impact?

Should not affect any part of the site and should be disabled: https://www.va.gov/benefit-eligibility-questionnaire

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Nice work! Thanks @vetskrieg

**Review by ghost — DISMISSED** [review]

**Review by IGallupSoCo — DISMISSED** [review]
LGTM

File: src/applications/benefit-eligibility-questionnaire/containers/App.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import RoutedSavableApp from 'platform/forms/save-in-progress/RoutedSavableApp';
+import formConfig from '../config/form';
+
+export default function App({ location, children }) {
+  return (
+    <>
+      <va-breadcrumbs uswds="false" class="va-nav-breadcrumbs">
```
Comment (powellkerry) [inline src/applications/benefit-eligibility-questionnaire/containers/App.jsx:9]: Is there a reason for not using the USWDS version of breadcrumbs?

File: src/applications/benefit-eligibility-questionnaire/containers/ConfirmationPage.jsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+          ) : null}
+
+          {isValid(submitDate) ? (
+            <p>
+              <strong>Date submitted</strong>
+              <br />
+              <span>{format(submitDate, 'MMMM d, yyyy')}</span>
+            </p>
+          ) : null}
+          <button
```
Comment (powellkerry) [inline src/applications/benefit-eligibility-questionnaire/containers/ConfirmationPage.jsx:55]: Should this be a va-button?

Comment (vetskrieg) [reply src/applications/benefit-eligibility-questionnaire/containers/ConfirmationPage.jsx]: This is updated to USWDS breadcrumb.

**Review by powellkerry — APPROVED** [review]

**Pull request merged**